src/bootstrap/tool.rs

```diff
@@ -218,7 +218,7 @@ pub fn prepare_tool_cargo(
     command: &'static str,
     path: &'static str,
 ) -> Command {
-    let mut cargo = builder.cargo(compiler, Mode::Tool, target, command);
+    let mut cargo = builder.cargo(compiler, Mode::ToolRustc, target, command);
```

Comment:
We'll want `prepare_tool_cargo` to take the mode as an argument; that'll make future changes easier. Generally speaking we should be threading tool modes through so that we never just pick "ToolRustc" for no reason.